ibis/tests/all/test_temporal.py

```diff
@@ -419,7 +416,6 @@ def test_day_of_week_column(backend, con, alltypes, df):
         ),
     ],
 )
-@pytest.mark.xfail_backends([PySpark])  # #2201
```

Comment:
Every so often when we change dependency versions, these tests flip between failing or not failing. Seems like whether these tests fail or not depends on the `pyspark` and `pyarrow` version combination.

Reply:
I think these are just like the UDF tests. Given that `pyspark` is `2.4.6`, these tests succeed if `pyarrow<=0.14.1` and fail otherwise.